src/elm/Vela.elm

```diff
@@ -865,21 +920,23 @@ buildUpdateRepoIntPayload field value =
 -- PIPELINE
 
 
-type alias Pipeline =
+type alias PipelineModel =
     { config : ( WebData PipelineConfig, Error )
     , expanded : Bool
     , expanding : Bool
     , org : Org
-    , repo : Org
+    , repo : Repo
+    , buildNumber : Maybe BuildNumber
```

Comment:
reasons for adding BuildNumber to PipelineModel will become more obvious, but this lets us tie it to the Build tabs in the next PR.